content/tutorials/quickstart/get-nftoken-list.md

```diff
@@ -0,0 +1,337 @@
+
+### Get NFTokens
+
+All transactions on the XRP Ledger are available for public review. You do not need an account or credentials to access information about minting and trading NFTokens.
+
+To get a list of current `NFTokens` on the XRP Ledger, click **Get NFTokens**. The button retrieves 200 transactions (the maximum batch allowance), then filters to show only the information from `NFToken` pages. The row marker for the response is stored in the **Marker** field. Each new search starts at the marker to return a unique set of records for the next 200 transactions. Click **Get NFTokens** repeatedly to retrieve all of the `NFTokens` on the XRP Ledger.
+
+### Get NFTokenIDs
+
+Click **Get NFToken IDs** for a more finely grained search that returns only the `NFTokenIDs` from each batch of 200 transactions. Continue to click **Get NFToken IDs**  to retrieve all `NFTokenIDs` on the XRP Ledger.
+
+
+### Get Info
+
+Having a list of `NFTokens` is all well and good, but it would be more interesting to do something with the `NFTokenIDs` returned. You can get information on the `NFTokens` in the response.
+
+To review `NFToken` information for a selected `NFTokenID`:
+
+1. Copy a `NFTokenID` from the list and paste it in the **NFTokenID** field.
+2. Click **Get NFToken Info**.
+
+
+# Code Walkthrough
+
+You can download the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip){.github-code-download} archive to try each of the samples in your own browser.
+
+## Get `NFTokens`
+
+Declare variables used in the function.
+
+```javascript
+async function getNFTokens () {
+	let jason
+	let ledger_entry_type
+	let nfTokens
+	let nftoken_data
+	let results = ""
+	let stateTree
+```
+
+Connect to the XLS-20Devnet server.
+
+```javascript
+  const client = new xrpl.Client('wss://xls20-sandbox.rippletest.net:51233')
+	await client.connect()
+```
+
+Check to see if the **Marker** field is populated with a 64 character value. If so, include the marker value in the `ledger_data` request. Set the limit of records to be returned to 200 transactions.
+
+```javascript
+  if (markerField.value.length == 64) {
+		nftoken_data = await client.request({
+			command: "ledger_data",
+			id: "NFToken data request",
+			limit: 200,
+			marker: markerField.value
+		})
+```
+
+Otherwise, send the `ledger_data` request without the marker field.
+
+```javascript
+
+  } else {
+		nftoken_data = await client.request({
+			command: "ledger_data",
+			id: "NFToken data request",
+			limit: 200
+		}) 
+  }
+```
+
+Turn the response into a JSON string.
+
+```javascript
+	jason =  JSON.stringify(nftoken_data.result, null, 2)
+```
+
+Parse the state tree from the JSON object.
+
+```javascript
+	stateTree = JSON.parse(jason).state
+```
```

Comment:
This looks to me like you are taking a JSON object, converting it to a string, and then back into a JSON object. Am I missing something here, or is this a completely unnecessary convert→unconvert step?